src/js/components/Box/StyledBox.js

```diff
@@ -67,32 +67,68 @@ const basisStyle = css`
 // https://stackoverflow.com/questions/36247140/why-doesnt-flex-item-shrink-past-content-size
 // we assume we are in the context of a Box going the other direction
 // TODO: revisit this
-const directionStyle = (direction, theme) => {
+const directionStyle = (directionWrapper, theme) => {
+  let direction;
+  let isResponsive;
+  let isReverse;
```

Comment:
What would be the expected behavior in case the user set the `direction` but hadn't set these two values?

Reply:
If user passes only direction like below:
```

<Box direction={
  direction: "row" // or column
}/>
```

then it will be non responsive like `<Box direction="row">` or `<Box direction="column">`

Reply:
Added story for above scenario.